config/config.go

```diff
@@ -888,6 +889,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.gumgum.endpoint", "https://g2.gumgum.com/providers/prbds2s/bid")
 	v.SetDefault("adapters.improvedigital.endpoint", "http://ad.360yield.com/pbs")
 	v.SetDefault("adapters.ix.endpoint", "http://appnexus-us-east.lb.indexww.com/transbidder?p=184932")
+	v.SetDefault("adapters.invibes.endpoint", "https://{{.Host}}/bid/ServerBidAdContent")
```

Comment:
Prebid Server is not a pass through service. We no longer accept using a publisher provided host and are working with existing bidders to fix their usage. There are other macros available such as account id. What part of your url is publisher dependent?

Reply:
The subdomain can differ. We currently have two production domains plus two testing domains.
Is it ok if instead of Host parameter, we add a DomainId parameter (of type integer) and select in code one domain or the other?

Reply:
Yes, I think that would be a good solution. In that case you may leave the `{{.Host}}` macro here and have the adapter choose the right one based on the DomainID parameter.